.github/workflows/validate_plugin.yml

```diff
@@ -0,0 +1,71 @@
+name: Plugin Validation
+
+on:
+  pull_request:
+    branches:
+      - "master"
+    paths:
+      - 'community-plugins.json'
+
+jobs:
+  plugin-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - run: npm install axios
+      - uses: actions/github-script@v5
+        with:
+          script: |
+            const fs = require('fs');
+            const axios = require('axios');
+            const author = context.payload.sender.login
+            const plugins = JSON.parse(fs.readFileSync('community-plugins.json', 'utf8'));
+            const plugin = plugins[plugins.length - 1];
+
+            const errors = [];
+            const addError = (error) => {
+              errors.push(`:x: ${error}\n`)
+            }
+            const generateMessage = () => {
+              let reply = `### Hello ${plugin.author}! <a href="https://obsidian.md"><img align="right" height="30px" src="https://user-images.githubusercontent.com/59741989/139557624-63e6e31f-e617-4041-89ae-78b534a8de5c.png"/></a>\n**I found the following Errors in your Plugin, ${plugin.name}:**\n\n`;
+              errors.forEach((e) => reply += e);
+              reply += `\n---\n<sup>This check was done automatically and might not be correct. If something went wrong, please ping \`@phibr0\`.`;
+              return reply;
+            }
+            
+            if (context.payload.pull_request.changed_files > 1) {
+                addError('You modified other Files.');
+            }
+
+            await axios.get(`https://api.github.com/repos/${plugin.repo}`)
+              .then(async () => {
```

Comment:
I think these then/catch chains can be simplified since you're already using await anyway.